.github/CHANGELOG.md

```diff
@@ -2,6 +2,66 @@
 
 <h3>New features since last release</h3>
 
+* Adds a decorator `@qml.qfunc_transform` to easily create a transformation
+  that modifies the behaviour of a quantum function.
+  [(#1315)](https://github.com/PennyLaneAI/pennylane/pull/1315)
+
+  For example, consider the following transform, which replaces all
+  `CRX` gates with a sequence of `RX`, `RY`, and `CZ` gates:
+
+  ```python
+  @qml.qfunc_transform
+  def my_transform(tape, x, y):
+      for op in tape.operations + tape.measurements:
+          if op.name == "CRX":
+              wires = op.wires
+              param = op.parameters[0]
+              qml.RX(x * param, wires=wires[1])
+              qml.RY(y * qml.math.sqrt(param), wires=wires[1])
+              qml.CZ(wires=[wires[1], wires[0]])
+          else:
+              op.queue()
```

Comment:
I still think it might be good to use an example that someone might actually want to do.  Like, scale all `RX` rotation angles by a parameter, or rewrite `CNOT` in terms of `CZ` and `H`.  I see this, and a part of my mind is trying to figure out "why this transformation" instead of just working on understanding the transform syntax.

Reply:
The scaling one is a nice one, I'll do that :)